.github/workflows/pr-checks.yml

```
@@ -0,0 +1,421 @@
+              repo: context.repo.repo,
+              comment_id: botComment.id,
+              body: summary
+            });
+          } else {
+            await github.rest.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              body: summary
+            });
+          }
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)